api/src/pcapi/connectors/dms/serializer.py

```diff
@@ -240,6 +240,10 @@ class ApplicationDetailNewJourney(ApplicationDetail):
     status: finance_models.BankAccountApplicationStatus
     label: str | None = None
 
+    @validator("label")
+    def truncate_label(cls: "ApplicationDetailNewJourney", label: str) -> str:
+        return label if len(label) <= 100 else label[:97] + "..."
```

Comment:
[micro-détail / question de goût / le saviez-vous ?] Il existe [textwrap.shorten()](https://docs.python.org/fr/3/library/textwrap.html#textwrap.shorten) dans la bibliothèque standard Python. :)  Ça donnerait ça : 

```suggestion
        return textwrap.shorten(label, 100, placeholder="...")
```

Ça tente de couper un peu plus élégamment en retirant des mots, jusqu'à que la chaîne et l'ellipse tiennent dans le nombre de caractères demandé.

(La troncature "manuelle" n'est pas déraisonnable, je ne rejette pas la PR pour autant. ;) )

Reply:
til !